kolibri/core/assets/src/views/SideNav.vue

```diff
@@ -201,9 +201,6 @@
       },
       handleClickPrivacyLink() {
         this.privacyModalVisible = true;
-        // HACK to fix https://github.com/learningequality/kolibri/issues/4973.
-        // For some reason, the button maintains focus even when the modal is opened.
```

Comment:
is this condition failing?

https://github.com/learningequality/kolibri/blob/develop/kolibri/core/assets/src/views/KModal.vue#L201

Reply:
I'm not sure. But even if you put the focus on the modal but clicking or tabbing on to it, then close it, you still get the same behavior only when the modal-opening button is an anchor tag. Might be specific just to `<a>`

Reply:
Yeah, I just checked it and it does enter into the focusModal branch when you click them (and this is without this PR).

Reply:
Yeah, I checked this with the Class Rename button, and it does not enter into the focusModal branch because, I guess the active element is still that link.

For the side-nav privacy link, it _does_ enter the focusModal branch because in that case the link was blurred in the hack we deleted here.